server/views/formPages/supervisor/review.html

```diff
@@ -0,0 +1,134 @@
+{% extends "../formTemplate.html" %}
+{% from "warning-text/macro.njk" import govukWarningText %}
+{% from "textarea/macro.njk" import govukTextarea %}
+{% from "button/macro.njk" import govukButton %}
+{% from "radios/macro.njk" import govukRadios %}
+
+{% set greenButtonText = "Continue" %}
+
+{% block formItems %}
+
+
+<div class="govuk-grid-row govuk-body-s">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-l">Review provisional categorisation</h1>
+
+    {% set cat = data.provisionCat %}
+    {% set overriddenCat = data.overCat %}
+    {% set currentCat %}{% if overriddenCat %}{{overriddenCat}}{% else %}{{cat}}{% endif %}{% endset %}
+
+
+    {% if overriddenCat %}
+    <div class="govuk-warning-text forms-info-text warning-overrides">
+      <span id="override-old-cat-icon" class="govuk-warning-text__icon" aria-hidden="true">{{ cat }}</span>
+      <span id="warning-arrow" class=""><img src='/assets/images/Triangle.png' alt='arrow' height='18'
+                                             width='12'/></span>
+      <span id="override-cat-icon" class="govuk-warning-text__icon" aria-hidden="true">{{ overriddenCat }}</span>
+      <strong id="override-icon-text" class="govuk-warning-text__text">
+        <span class="govuk-warning-text__assistive">Warning</span>
+        The category was originally {{ cat }} and is now {{ overriddenCat }}
+      </strong>
+    </div>
+    {% else %}
+    <div class="govuk-warning-text forms-info-text warning-overrides">
+      <span class="govuk-warning-text__icon" aria-hidden="true">{{ cat }}</span>
+      <strong class="govuk-warning-text__text">
+        <span class="govuk-warning-text__assistive">Warning</span>
+        Based on the information provided, the provisional category is {{ cat }}
+      </strong>
+    </div>
+    {% endif %}
+
+
+    {% set showChangeLinks %}false{% endset %}
+    {% include "../../partials/reviewContents.html" %}
+
+
+    {% set overriddenCategoryTextHtml %}
+
+    {% if (currentCat == "B" or currentCat == "C" or currentCat == "D") %}
+      {{ govukRadios({
```

Comment:
this is shared by the provisional category page and will be moved into a partial